CMakeLists.txt

```
@@ -284,6 +284,10 @@ elseif(UNIX)
     )
     target_compile_definitions(PortAudio PUBLIC PA_USE_COREAUDIO=1)
     set(PKGCONFIG_CFLAGS "${PKGCONFIG_CFLAGS} -DPA_USE_COREAUDIO=1")
+
+    # Use C11 so that we can make use of atomic library and avoid deprecation errors.
+    target_compile_definitions(PortAudio PRIVATE std=c11)
```

It looks like you mixed up a compile definition with a compiler option. Anyway, this can be done in a more generic way using CMake's C_STANDARD feature:
```suggestion
    set_property(TARGET PortAudio PROPERTY C_STANDARD 11)
```

---

Thanks!